Controller/src/main/java/org/apidb/irods/BuildEventsFile.java

```diff
@@ -22,82 +19,106 @@
  * the entire iRODS events folder, and consolidates them into a single JSON
  * array of events that is passed on to the WDK's user dataset events handler
  * for processing according to the nature of each event.
- *
+ * <p>
  * Although this project is specifically meant to address event handling
  * intended for iRODS, it has no specific references to iRODS.  It could be used
  * with a POSIX system as well.
  *
  * @author crisl-adm
  */
-public class BuildEventsFile {
+public class BuildEventsFile
+{
 
-  private static final Logger logger = Logger.getLogger(BuildEventsFile.class);
-  public static final String EVENTS_DIR = "/ebrc/workspaces/events";
+  private static final Logger LOG        = Logger.getLogger(BuildEventsFile.class);
+  public static final  String EVENTS_DIR = "/ebrc/workspaces/events";
 
   public static void main(String[] args) throws Exception {
 
-	// The id of the project for which these events are intended.
-    String projectId = System.getenv("PROJECT_ID");
+    // The id of the project for which these events are intended.
+    var projectId = System.getenv("PROJECT_ID");
 
     // Identifies the datastore associated with the events to be handled.
     // Used to insure that this datastore matches the one this build supports.
-    String datasetStoreId = System.getenv("DATASET_STORE_ID");
+    var datasetStoreId = System.getenv("DATASET_STORE_ID");
```

Comment:
How does a datastore get an ID?  is this more like an address?  if so, consider changing the name of DATA_STORE_ID.   this code is begging the quesion of where do these IDs come from and what do they look like, even tho, in a sense that is a concern of the caller

Reply:
I don't actually know how the caller gets this value, I assume it's part of the iRODS config or something.  It's not an address, it's an identifier string (one that I don't think should be posted to github?).  The string appears to be the name of the iRODS instance or something similar, @rbelnap can probably provided more info.

Reply:
@steve-fischer-200 the dataset_store_id was implemented by Cris at your request. I believe the intent was to provide a check for situations where the handler is misconfigured and pointed to a different irods (i.e. prod pointing to a dev irods). The value is stored in irods as metadata, and passed into jenkins as a parameter. I believe the handler then doesn't function if they don't match? I'm not sure how well this actually prevents a misconfiguration issue, since the parameter can also be misconfigured to match... but this is why it exists.